src/navigation/tabs/tabs.jsx

```diff
@@ -1,86 +1,668 @@
+import DropdownButton from '../../inputs/dropdownButton';
+import TabsTab from './tabsTab';
+import TabsContent from './tabsContent';
+import withStyles from '../../styles/withStyles';
 
-class Tabs extends React.Component {
+const PREFIX_TAB = 'tab-';
+const CONTENT_PREFIX = 'content-';
+const HIDDEN_TABS_ICON_WIDTH = 30;
+const WIDTH_OFFSET = 5; // it ain't neccesary to be too precise when resizing and calculating hidden tabs
+
+const propTypes = {
+    /**
+     * beforeChange callback: return false to prevent tab change
+     */
+    beforeChange: PropTypes.func,
+    /**
+     * Custom classes to override the default styling
+     */
+    classes: PropTypes.shape({
+        inverse: PropTypes.string,
+        label: PropTypes.string,
+        mobile: PropTypes.string,
+        moreButton: PropTypes.string,
+        moreButtonInnerContainer: PropTypes.string,
+        root: PropTypes.string,
+        selected: PropTypes.string,
+        tab: PropTypes.string,
+        tabsPanel: PropTypes.string,
+        tabsPanelContent: PropTypes.string,
+        withContent: PropTypes.string,
+    }),
+    /**
+     * If `true`, Tabs will be formatted to appear on dark backgrounds.
+     */
+    inverse: PropTypes.bool,
+    /**
+     * List of the tabs objects
+     */
+    items: PROP_TYPES_ITEMS,
+    /**
+     * If `true`, Tabs will look more like actionable buttons.
+     */
+    mobile: PropTypes.bool,
+    /**
+     * onChange active tab callback
+     */
+    onChange: PropTypes.func,
+    /**
+     * Resize recalculation frecuency in milliseconds
+     */
+    resizeThrottle: PropTypes.number,
+    /**
+     * Use this prop to programatically change the selected tab
+     */
+    selectedTabKey: PROP_TYPES_SELECTED_TAB_KEY,
+    /**
+     * Renders the content set inside the item object under the tabs panel
+     */
+    withContent: PropTypes.bool,
+};
+
+const defaultProps = {
+    beforeChange: undefined,
+    classes: null,
+    inverse: false,
+    items: [],
+    mobile: false,
+    onChange: undefined,
+    resizeThrottle: 100,
+    selectedTabKey: undefined,
+    withContent: false,
+};
+
+const styles = ({ palette, spacing }) => ({
+    mobile: {},
+    moreButton: {
+        height: '26px !important',
+        margin: 0,
+        minHeight: '26px !important',
+        padding: 0,
+        '&.button_dropdown-open': {
+            '& .icon-use-path': {
+                fill: `${palette.cyan[500]} !important`,
+            },
+        },
+        '& .ui.icon': {
+            marginTop: '-4px !important',
+        },
+    },
+    moreButtonInnerContainer: {
+        minHeight: 26,
+    },
+    root: {
+        margin: [[0, -spacing(1)]],
+        position: 'relative',
+        '&$mobile': {
+            margin: [[0, -4]],
+        },
+    },
+    tabsPanel: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        '&$withContent': {
+            marginBottom: -1,
+            position: 'relative',
+            zIndex: 5,
```

Comment:
This props is introducing an issue with the <AppBar /> (wich zIndex value is 1).

![image](https://user-images.githubusercontent.com/8116485/100165050-ddfc7980-2e7e-11eb-8c6d-9c4f2b0639ba.png)

What could be the right zIndex value for the <AppBar />? Let me know if you want me to fix this thing in the app bar side. @cbrewer1689